crates/shared/src/conversions.rs

```diff
@@ -1,7 +1,77 @@
 use anyhow::{ensure, Result};
+use database::orders::{
+    BuyTokenDestination as DbBuyTokenDestination, OrderKind as DbOrderKind,
+    SellTokenSource as DbSellTokenSource, SigningScheme as DbSigningScheme,
+};
+use model::{
+    order::{BuyTokenDestination, OrderKind, SellTokenSource},
+    signature::SigningScheme,
+};
 use num::{rational::Ratio, BigInt, BigRational};
 use primitive_types::U256;
 
+pub fn order_kind_into(kind: OrderKind) -> DbOrderKind {
+    match kind {
+        OrderKind::Buy => DbOrderKind::Buy,
```

Comment:
Could we put the functions you moved into this file into a new module like "db_order_conversions"? I would like to keep these and what was already in this file separate.